src/mp4box/stts.rs

```diff
@@ -55,6 +55,11 @@ impl<R: Read + Seek> ReadBox<&mut R> for SttsBox {
         let (version, flags) = read_box_header_ext(reader)?;
 
         let entry_count = reader.read_u32::<BigEndian>()?;
+        if u64::from(entry_count) > size.saturating_sub(4) / 8 {
```

Comment:
Still need to document these.

Reply:
looks addressed by [1c8ca74](https://github.com/privacyresearchgroup/mp4-rust/pull/2/commits/1c8ca74b93664391d1c1441ec45bf36b6b74e690)